modules/ROOT/pages/dataweave-pattern-matching.adoc

```diff
@@ -97,9 +97,11 @@ output application/json
 }
 ----
 
-For use cases where you need a boolean result based on whether a value
-matches or not, you can use the `matches` function instead of using pattern
-matching.
+For use cases in which the input is a String, you can also use the  xref:dw-core-functions-match.adoc[match] _function_ in the `dw::Core` module to test the string against a regex pattern. 
+
+For use cases where you need a boolean result based on whether a String left-side input argument matches some regex pattern, you can use the `matches` function instead of using pattern matching or the dw::Core module's `match` function.
```

Comment:
```suggestion
For use cases that require a Boolean result based on whether an input argument matches a regex pattern, you can use the  xref:dw-core-functions-matches.adoc[matches] _function_ in the `dw::Core` module instead of using pattern matching or the `match` function.
```